src/utils/internalLinks.js

```diff
@@ -218,3 +219,14 @@ export const getMessageIdFromLink = (url: string, realm: URL): number => {
 
   return isMessageLink(url, realm) ? parseInt(paths[paths.lastIndexOf('near') + 1], 10) : 0;
 };
+
+export const getStreamTopicURL = (
```

Comment:
nit: apply camel case even to words that would be all-caps in prose. So here, `getStreamTopicUrl`.

That's what we generally do; you can see some other examples in this file. (Grepping more broadly, I guess we didn't on `AvatarURL` and its subclasses, but we're consistent basically everywhere else.)

The Google style guides have a formal description of this capitalization scheme: https://google.github.io/styleguide/jsguide.html#naming-camel-case-defined